docs/compose.md

```diff
@@ -37,10 +37,32 @@ functionality that you don't require.
 Currently the customization is limited as described below but future
 releases will increase the amount of customization that is available.
 It is also possible to [build Triton](build.md#building-triton)
-yourself to get more exact customization.
+from source to get more exact customization.
 
-The two Docker images used for customization are retrieved using the
-following commands.
+## Use the compose.py script
+
+The `compose.py` script uses the existing NGC Docker images 
+```
+nvcr.io/nvidia/tritonserver:<xx.yy>-py3-min
+nvcr.io/nvidia/tritonserver:<xx.yy>-py3
+```
+to create the new custom Triton Docker image.
```

Comment:
describe here how you need to "git clone" the release branch of the server repo that corresponds to the docker container release that you want to use to compose a new image

Reply:
Would it not be easier to ask the user specify the upstream container version and if they don't specify, then fallback to branch specific version? This way, people don't have to clone the entire repo just for this script

Reply:
We can make --container-version optional. The potential problem is that the branch they run from (e.g. "main") may have changes in compose.py that are not valid for the release container they target. So the default should be to work using TRITON_VERSION but it is ok to have --container-version flag to override if the user knows what they are doing.